nf_research_tools.rdb.model.csv

```diff
@@ -11,7 +11,8 @@ Usage Requirements,Any known restrictions on use of the resource. ,"non-commerci
 How to Acquire,How to acquire a particular resource. ,,,TRUE,,Resource,,,
 Date Added,The date that the resource was originally added.,,,TRUE,,Resource,,,
 Date Modified,The last update of the resource metadata (not including observations).,,,TRUE,,Resource,,,
-Genetic Reagent,Genetic reagents.,,"Component,  Genetic Reagent_id, Insert Name, Insert Entrez_id, gRNA shRNA sequence, Insert Size, Insert Species, nTerminal Tag, cTerminal Tag, Cloning Method, 5prime Cloning Site, 5prime Site Destroyed, 3prime Clonin gSite, 3prime Site Destroyed, Promoter, 5primer, 3primer, Vector Backbone, Vector Type, Backbone Size, Total Size, Bacterial Resistance, Selectable Marker, Copy Number, Growth Temp, Growth Strain, Hazardous",FALSE,,Genetic Reagent,"Genetic Reagent_id, Insert Name, Insert Entrez_id, gRNA shRNA sequence, Insert Size, Insert Species, nTerminal Tag, cTerminal Tag, Cloning Method, 5prime Cloning Site, 5prime Site Destroyed, 3prime Clonin gSite, 3prime Site Destroyed, Promoter, 5primer, 3primer, Vector Backbone, Vector Type, Backbone Size, Total Size, Bacterial Resistance, Selectable Marker, Copy Number, Growth Temp, Growth Strain, Hazardous",,
+Genetic Reagent,Genetic reagents.,,"Component,  Genetic Reagent_id, Insert Name, Insert Entrez_id, gRNA shRNA sequence, Insert Size, Insert, nTerminal Tag, cTerminal Tag, Cloning Method, 5prime Cloning Site, 5prime Site Destroyed, 3prime Clonin gSite, 3prime Site Destroyed, Promoter, 5primer, 3primer, Vector Backbone, Vector Type, Backbone Size, Total Size, Bacterial Resistance, Selectable Marker, Copy Number, Growth Temp, Growth Strain, Hazardous",FALSE,,Genetic Reagent,"Genetic Reagent_id, Insert Name, Insert Entrez_id, gRNA shRNA sequence, Insert Size, Insert Species, nTerminal Tag, cTerminal Tag, Cloning Method, 5prime Cloning Site, 5prime Site Destroyed, 3prime Cloning Site, 3prime Site Destroyed, Promoter, 5primer, 3primer, Vector Backbone, Vector Type, Backbone Size, Total Size, Bacterial Resistance, Selectable Marker, Copy Number, Growth Temp, Growth Strain, Hazardous",,

```

Comment:
```suggestion
Genetic Reagent,Genetic reagents.,,"Component,  Genetic Reagent_id, Insert Name, Insert Entrez_id, gRNA shRNA sequence, Insert Size, Insert Species, nTerminal Tag, cTerminal Tag, Cloning Method, 5prime Cloning Site, 5prime Site Destroyed, 3prime Clonin gSite, 3prime Site Destroyed, Promoter, 5primer, 3primer, Vector Backbone, Vector Type, Backbone Size, Total Size, Bacterial Resistance, Selectable Marker, Copy Number, Growth Temp, Growth Strain, Hazardous",FALSE,,Genetic Reagent,"Genetic Reagent_id, Insert Name, Insert Entrez_id, gRNA shRNA sequence, Insert Size, Insert Species, nTerminal Tag, cTerminal Tag, Cloning Method, 5prime Cloning Site, 5prime Site Destroyed, 3prime Cloning Site, 3prime Site Destroyed, Promoter, 5primer, 3primer, Vector Backbone, Vector Type, Backbone Size, Total Size, Bacterial Resistance, Selectable Marker, Copy Number, Growth Temp, Growth Strain, Hazardous",,

```

Reply:
Oops, I missed this first time around. Was "species" removed by accident here?

Reply:
Ah maybe by accident!